diesel/src/associations/belongs_to.rs

```diff
@@ -131,6 +241,36 @@ where
         }
         result
     }
+
+    fn try_grouped_by(
+        self,
+        parents: &'a [Parent],
+    ) -> Result<Vec<Vec<Child>>, TryGroupedByError<Child>> {
+        use std::{collections::HashMap, iter};
```

Comment:
```suggestion
        use std::collections::HashMap;
        use std::iter;
```

to keep this consistent with the other imports in diesel.